src/nethsec/firewall/__init__.py

```diff
@@ -524,3 +540,109 @@ def disable_ipv6_firewall(uci):
 
     uci.save("firewall")
     return disabled
+
+
+def list_zones(uci) -> dict:
+    """
+    Get all zones from firewall config
+
+    Args:
+        uci: EUci pointer
+
+    Returns:
+        dict with all zones
+    """
+    return utils.get_all_by_type(uci, 'firewall', 'zone')
+
+
+def list_forwardings(uci) -> dict:
+    """
+    Get all forwardings from firewall config
+
+    Args:
+        uci: EUci pointer
+
+    Returns:
+        dict with all forwardings
+    """
+    return utils.get_all_by_type(uci, 'firewall', 'forwarding')
+
+
+def __add_forwarding(uci, src: str, dest: str):
+    """
+    Add forwarding from src to dest.
+
+    Args:
+        uci: EUci pointer
+        src: source zone
+        dest: destination zone
+    """
+    config_name = PREFIX_CONFIG + src + '2' + dest
```

Comment:
Use the get_id function
```suggestion
    config_name = utils.getid(src + '2' + dest)
```